controller.js

```diff
@@ -43,11 +44,37 @@ exports.getArticle = (req, res, next) => {
 };
 
 exports.getArticles = (req, res, next) => {
-  selectAllArticles().then((allArticles) => {
-    addComments(allArticles).then((allArticlesWithComments) => {
-      res.status(200).send(allArticlesWithComments);
+  const queries = Object.keys(req.query);
+
+  let sort_by = undefined;
+  let order = undefined;
+
+  if (queries.includes("sort_by")) {
+    sort_by = req.query.sort_by;
+  }
+  if (queries.includes("order")) {
+    order = req.query.order;
+  }
+
+  if (sort_by !== undefined || order !== undefined) {
```

Comment:
All of this logic from lines 47 to 59 would be nicer in your model. We want the controller to just essentially be calling the model and sending the response 👍